--- Cython-0.23/Cython/Compiler/PyrexTypes.py.orig	2016-02-05 07:56:42.411235179 -0500
+++ Cython-0.23/Cython/Compiler/PyrexTypes.py	2016-02-05 07:59:35.926182417 -0500
@@ -1960,7 +1960,8 @@
     #define __Pyx_CIMAG(z) ((z).imag)
 #endif
 
-#if (defined(_WIN32) || defined(__clang__)) && defined(__cplusplus) && CYTHON_CCOMPLEX
+#if defined(__cplusplus) && !CYTHON_COMPLEX \
+        && (defined(_WIN32) || defined(__clang__) || (defined(__GNUC__) && GCC_VERSION >= 40400) || __cplusplus >= 201103)
     #define __Pyx_SET_CREAL(z,x) ((z).real(x))
     #define __Pyx_SET_CIMAG(z,y) ((z).imag(y))
 #else
